cli/pkg/kctrl/cmd/package/installed/create_or_update.go

```diff
@@ -159,10 +159,8 @@ func NewUpdateCmd(o *CreateOrUpdateOptions, flagsFactory cmdcore.FlagsFactory) *
 		cmd.Args = cobra.ExactArgs(1)
 	}
 
-	cmd.Flags().StringVarP(&o.packageName, "package", "p", "", "Name of package install to be updated")
```

Comment:
not come package is removed? it's valid to change PackageInstall to use a different package name. (though probably less common). may be it should be --dangerous-package?

Reply:
Hmm, makes sense. I will revert this commit for now. 
cc @100mik